google/cloud/automl_v1beta1/services/tables/tables_client.py

```diff
@@ -1524,7 +1576,7 @@ def set_time_column(
             column_spec_display_name=column_spec_display_name,
             project=project,
             region=region,
-            **kwargs
+            **kwargs,
```

Comment:
A continuation to my above comment, the kwargs in the code above (line 1285) is deleted, but here is kept. It's better to be consistent.